wire-ios-transport/Source/TransportEncoding/ZMTransportCodec.m

```diff
@@ -24,10 +24,11 @@ @implementation ZMTransportCodec
 
 + (id<ZMTransportData>)interpretResponse:(NSHTTPURLResponse *)response data:(NSData *)data error:(NSError *)error;
 {
-    if ((data == nil) || (error != nil) || response.statusCode >= 500) {
+    /// We will try to serialize the data and get the payload despite the error and response.statusCode.
```

Comment:
I don't think this comment is necessary because it refers to code that is no longer present.

Reply:
Done